crates/bevy_reflect/src/type_info.rs

```diff
@@ -46,9 +46,11 @@ use std::any::{Any, TypeId};
 ///   }
 /// }
 ///
+/// # impl TypeName for MyStruct {
+/// #   fn name() -> std::borrow::Cow<'static, str> { todo!() }
```

Comment:
I think this is supposed to be (+ with an actual return value):

```suggestion
/// #   fn name() -> &'static str { "MyStruct" }
```